src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -62,8 +62,8 @@ private TypicalPersons() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the typical persons.
      */
-    public static AddressBook getTypicalAddressBook() {
-        AddressBook ab = new AddressBook();
+    public static Scheduler getTypicalAddressBook() {
+        Scheduler ab = new Scheduler();
```

Comment:
Variable name
`ab` was meant for AddressBook

Reply:
Will be doing next